cfme/tests/cloud/test_security_groups.py

```diff
@@ -0,0 +1,61 @@
+import fauxfactory
+import pytest
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils.wait import TimedOutError
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.provider([OpenStackProvider], scope="module")
+]
+
+
+@pytest.yield_fixture(scope='module')
+def sec_group(appliance, provider):
+    security_group = appliance.collections.security_group
```

Comment:
**Required**

Please switch the pluralization here - `security_group` (singular) is actually a collection, and `sec_groups` (plural) is actually an entity.

Makes further calls, like `sec_groups.delete()` confusing, as it looks like a call to the collection delete.

Reply:
I think it should also include col or collection in name so that we know it's not just a list of them